package.json

```diff
@@ -131,6 +131,7 @@
     "@dlc-link/dlc-tools": "1.1.1",
     "@fungible-systems/zone-file": "2.0.0",
     "@hirosystems/token-metadata-api-client": "1.2.0",
+    "@leather-wallet/tokens": "0.0.5",
```

Comment:
I made a mistake here installing this as a dependancy rather than a dev dependancy. 

I think it's OK to release as is but I can fix it